deploy.py

```diff
@@ -10,21 +10,33 @@
 from urllib.error import HTTPError
 from copy import deepcopy
 import re
+import ssl
 
 # UID for the folder under which our dashboards will be setup
 DEFAULT_FOLDER_UID = '70E5EE84-1217-4021-A89E-1E3DE0566D93'
 
-def grafana_request(endpoint, token, path, data=None):
+
+def grafana_request(endpoint, token, path, data=None, no_tls_verify=False):
     headers = {
         'Authorization': f'Bearer {token}',
         'Content-Type': 'application/json'
     }
     method = 'GET' if data is None else 'POST'
     req = Request(f'{endpoint}/api{path}', headers=headers, method=method)
+
     if not isinstance(data, bytes):
         data = json.dumps(data).encode()
-    with urlopen(req, data) as resp:
-        return json.load(resp)
+
+    if no_tls_verify:
+        ctx = ssl.create_default_context()
+        ctx.check_hostname = False
```

Comment:
Can you move the conditional check for no_tls_verify here? That way, we pass the context unconditionally to urlopen, and don't have to repeat the `urlopen` in line 35:36